commands/setup/setup.go

```diff
@@ -18,15 +18,36 @@
 package setup
 
 import (
-	"fmt"
+	"github.com/apache/skywalking-infra-e2e/internal/components/setup"
+	"github.com/apache/skywalking-infra-e2e/internal/logger"
 
 	"github.com/spf13/cobra"
+
+	"github.com/apache/skywalking-infra-e2e/internal/flags"
 )
 
+func init() {
+	Setup.Flags().StringVar(&flags.Env, "env", "kind", "specify the run environment")
+	Setup.Flags().StringVar(&flags.File, "file", "kind.yaml", "specify configuration file")
+}
+
 var Setup = &cobra.Command{
 	Use:   "setup",
 	Short: "",
 	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Change this kind of methods to `RunE: func(cmd *cobra.Command, args []string) error {` so that we can deal the `error` properly, do not `panic` or `logger.Log.Fatal` at a deep level of the codes and give possibilities to the upper functions to deal with the errors

Reply:
FYI @fgksgf 👆, let's propagate the errors to the up-most command

Reply:
I will change commands into that form.